content/intro-to-storybook/react/fr/creating-addons.md

```diff
@@ -0,0 +1,468 @@
+---
+title: 'Bonus : créer un addon'
+tocTitle: "Bonus : création d'addons"
+description: "Apprenez à construire vos propres addons qui vous permettront d'optimiser votre développement"
+commit: 'bebba5d'
+---
+
+Plus tôt, nous avons lancé une fonctionnalité clé du Storybook : le robuste écosystème [addons](https://storybook.js.org/addons/introduction/). Les addons sont utilisés pour améliorer l'expérience et les workflows des développeurs.
+
+Dans ce chapitre bonus, nous allons voir comment nous créons notre propre addon. Vous pensez peut-être que l'écrire peut être une tâche intimidante, mais en fait, ce n'est pas le cas, nous devons juste prendre quelques mesures pour commencer et nous pouvons commencer à l'écrire.
+
+Mais d'abord, il faut d'abord déterminer ce que notre addon va faire.
+
+## L'addon que nous allons écrire
+
+Pour cet exemple, supposons que notre équipe dispose de certains design assets qui sont d'une manière ou d'une autre liés aux composantes existantes de l'interface utilisateur. En examinant l'interface utilisateur actuelle de Storybook, il semble que cette relation ne soit pas vraiment apparente. Comment pouvons-nous y remédier?
+
+Nous avons notre objectif, maintenant définissons les fonctionnalités que notre addon va prendre en charge:
+
+- Affichage du design assets dans un panneau
```

Comment:
Affichage de la maquette dans un "panneau"

Reply:
Should I replace all "design assets" with "maquette"?